brands/shop/shoes.json

```diff
@@ -963,5 +963,16 @@
       "name:ja": "東京靴流通センター",
       "shop": "shoes"
     }
+  },
+  "shop/shoes|Dr. Martens": {
+    "countryCodes": ["uk"],
```

Comment:
I'd remove this line, since they operate retail stores in places other than just the `uk`.
(also, a thing that we get wrong sometimes, the [ISO country code](https://www.nationsonline.org/oneworld/country_code_list.htm) for uk is actually `gb`)

Reply:
These shops are actually all over so you can just remove it entirely